src/storage.ts

```diff
@@ -1,5 +1,6 @@
 import * as core from 'webcrypto-core'
 import { type Key as AskarKey, KeyAlgs } from '@hyperledger/aries-askar-shared'
+import type { KeyType, KeyUsage } from './types'
 
 const keyStorage = new WeakMap<core.CryptoKey, AskarKey>()
```

Comment:
Why do we need this? Can't we fetch the askar key based on e.g. the public key? This will be empty every time your start the server, so I don't understand the purpose of it.